lenet/README.md

```diff
@@ -0,0 +1,13 @@
+## WebNN API LeNet Example
+The sample uses the LeNet classifications network as an example.
```

Comment:
It would be good to show a diagram of the LeNet topology with the node names match the code, so readers can follow the code easily.

Reply:
That's a good idea. I'll add it.

Reply:
fixed it by adding a table that lists all nodes. See https://github.com/huningxin/webnn-samples/blob/lenet/lenet/README.md